.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+OpenMS Documentation Contributing Guidelines
+============================================
+
+Hi there!
+
+Thank you for thinking about enhancing OpenMS Documentation further.
+
+Please feel free to:
+
+1. Create a bug report or feature request in OpenMS Documentation, [here](https://github.com/OpenMS/OpenMS-docs/issues).
+2. Create a pull request in [OpenMS Documentation](https://github.com/OpenMS/OpenMS-docs) with the change you're proposing.
+
+For any questions, drop us a ping in [OpenMS Gitter](https://gitter.im/OpenMS/OpenMS).
+
+## Create a Pull Request(PR)
+
+1. Fork this repository.
+2. Add the change in your fork.
+3. Create a pull request to [OpenMS/OpenMS-Docs](https://github.com/OpenMS/OpenMS-docs/tree/staging) `staging` branch.
+4. Make sure you attach a few screenshots as to how your change looks like.
+5. If this change belongs to [OpenMS API reference](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/index.html),
+   please create a pull request [here](https://github.com/OpenMS/OpenMS/tree/develop/doc).
+
+## Documentation content writing conventions
+
+1. Please prefer active voice.
+2. Restrict the line length to 120 characters, including space(s).
+3. Title and H1 heading should be in `Title Case`. Follow `Sentence case`, until otherwise stated.
+4. Write as OpenMS is writing for itself as an object and subject.
+5. Use American English.
+6. Use `backtick`(s) for formatting code, library name, files, and path.
+7. Use **bold** for product name, object name, an independent entity.
+8. Use **bold** for menu title in an application.
+9. Link to glossary terms using {term}`this is a glossary term`.
```

Comment:
is this glossary thing working? this line looks just like normal text. no link, tooltip, or even changed color

Reply:
(this is an example glossary term formatting.)

Reply:
Shouldn't we then use a verbatim environment so that doc contributors see the exact code that they need to use?